tests/foreman/ui/test_contentview.py

```diff
@@ -847,11 +846,45 @@ def test_negative_add_dupe_modules(self):
 
         @assert: User cannot add modules multiple times to the view
 
-        @caseautomation: notautomated
-
-
         @CaseLevel: Integration
         """
+        cv_name = gen_string('alpha')
+        module_name = 'samba'
```

Comment:
@ldjebran could we perhaps create a small function that'd list a random module from a given repo? That would make us independent on what the repo actually contains.